cmd/duffle/relocate.go

```diff
@@ -106,18 +107,14 @@ func (r *relocateCmd) run() error {
 		return err
 	}
 
-	return r.writeBundle(bun)
+	return nil
```

Comment:
This whole section (lines 106-110) should be abbreviated to `return r.relocate(bun)`.

Reply:
Yup - another left-over I forgot to clean up. The linter didn't spot it.